build.gradle

```diff
@@ -130,6 +130,7 @@ dependencies {
     compile "org.jetbrains.kotlin:kotlin-stdlib-common:${kotlin_version}"
     compile 'org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.7'
     compile "org.jetbrains:annotations:13.0"
+    compile "org.jetbrains.kotlin:kotlin-stdlib-jdk8:${kotlin_version}"
```

Comment:
Seems like you mentioned this is just for one method. Possible to pull in that method ourselves instead of adding a whole dependency?

Reply:
make sense, copied that method from kotlin stdlib to our RestHandlerUtils